apps/src/templates/sectionAssessments/SectionAssessments.jsx

```diff
@@ -59,25 +59,34 @@ class SectionAssessments extends Component {
             scriptId={scriptId}
             onChange={this.onChangeScript}
           />
+          {!isLoading &&
+            <div>
+              <div style={{...h3Style, ...styles.header}}>
+                {i18n.selectAssessment()}
+              </div>
+              <AssessmentSelector
+                assessmentList={assessmentList}
+                assessmentId={assessmentId}
+                onChange={this.props.setAssessmentId}
+              />
+            </div>
+          }
         </div>
-        <div>
-          <div style={{...h3Style, ...styles.header}}>
-            {i18n.selectAssessment()}
+        {!isLoading &&
+          <div>
+            {/* Assessments */}
+            <MCAssessmentsOverviewContainer />
+            <StudentsMCSummaryContainer />
+            <MultipleChoiceByStudentContainer />
```

Comment:
Jist because I'm curious: Why do some of the components have `MultipleChoice` and some abbreviate to `MC`?

Reply:
We might want to consider abbreviating all of them for consistency - but the `MC` ones got abbreviated because the component names seemed very over the top length wise.